migration/v0.30/apply/30-remove-starboard.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    for CLUSTER in sc wc; do
+    log_info "Uninstall starboard"
+      helm_uninstall ${CLUSTER} monitoring starboard-operator
+      helm_uninstall ${CLUSTER} monitoring starboard-psp-rbac
+      log_info "Delete starboard crds"
+
+      STAR_CRDS=$(
+        kubectl_do ${CLUSTER} api-resources --api-group=aquasecurity.github.io -o name
+      )
+      if [ -n "$STAR_CRDS" ]; then
+          # shellcheck disable=SC2086
+          kubectl_do ${CLUSTER} delete crds --ignore-not-found=true $STAR_CRDS
```

Comment:
@aarnq I just realized that this is not super good, as running this again will remove the new CRDs aswell :disappointed:
I guess i must check the actual CRD version and if that is the old version delete it...

Reply:
Fixed